test/libsolidity/semanticTests/builtinFunctions/blobhash.sol

```diff
@@ -0,0 +1,17 @@
+contract C {
+    function f() public view returns(bytes32) {
+        return blobhash(0);
+    }
+    function g() public view returns(bytes32) {
+        return blobhash(1);
+    }
+    function h() public view returns(bytes32) {
+        return blobhash(2);
+    }
+}
+// ====
+// EVMVersion: >=cancun
+// ----
+// f() -> 0x0100000000000000000000000000000000000000000000000000000000000001
+// g() -> 0x0100000000000000000000000000000000000000000000000000000000000002
+// h() -> 0x00
```

Comment:
Is this returning zero because there are only two values defined in EVMHost? If so, leaving a comment above `h()` to indicate that this is the case would be nice.

Reply:
Yes. I added a comment to make it clear in the test.